R/rbabylon.R

```diff
@@ -1,14 +1,36 @@
 #' @name rbabylon
 #' @title RBabylon
-#' @description RBabylon is a library written in R that facilitates modeling and simulation by exposing functionality from the https://github.com/metrumresearchgroup/babylon project in R
-#' From Babylon's docs: Babylon is (will be) a complete solution for managing projects involving modeling and simulation with a number of software solutions used in pharmaceutical sciences.
-#' This is a fork of the `nonmemutils`` project that is broader in scope.
-#' Initial support encompasses NONMEM however the api is designed in a way to be flexible to handle other software.
+#' @description RBabylon is a library written in R that facilitates modeling and
+#'   simulation by exposing functionality from the
+#'   https://github.com/metrumresearchgroup/babylon project in R. From Babylon's
```

Comment:
Consider making this a hyperlink, as described [here](https://cran.r-project.org/web/packages/roxygen2/vignettes/rd-formatting.html).